Web/starter_project_vanguard/pages/profile/index.tsx

```diff
@@ -0,0 +1,62 @@
+import Link from 'next/link'
+import React, { useState } from 'react'
+
+import AccountSetting from '@/components/profile/AccountSetting'
+import PersonalInfo from '@/components/profile/PersonalInfo'
+
+const Profile: React.FC = () => {
+  const [tab, setTab] = useState<number>(0)
+  const handleTabChange = (tab: number) => {
```

Comment:
[nit] is there maybe a way to use prebuilt tabs for this? it would make things cleaner if possible but not mandatory for now